cdk/bin/cdk-pipeline-codu.ts

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+import "source-map-support/register";
+import * as cdk from "aws-cdk-lib";
+import { PipelineStack } from "../lib/pipeline-stack";
+
+const app = new cdk.App();
+
+new PipelineStack(app, "CoduPipeline", {
+  env: {
+    region: "eu-west-1",
```

Comment:
Move these to env variables

Reply:
These aren't secrets weirdly but I wanted to obfuscate it so I just figured out how. 

<3